routing/probability_estimator_test.go

```diff
@@ -104,12 +118,25 @@ func TestProbabilityEstimatorOneSuccess(t *testing.T) {
 		testTime, node1, 100, testCapacity, aprioriPrevSucProb,
 	)
 
+	// A high success probability is not influcenced by the capacity factor.
```

Comment:
Not sure I understand what's been tested here.

Reply:
I updated the comment, hope it's more clearer now